### DISCUSSION TITLE
Welcome to GarHal_CSGO Discussions!

### DISCUSSION THREAD

**Discussion by dretax** [dretax/GarHal_CSGO#83] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gartand** [comment]
Very good thing thank you. Very useful. Nice name.

**Reply by dretax** [reply]
Every good project comes with a meme name ;)
